src/options/common.ts

```diff
@@ -96,6 +96,23 @@ export const PUG_BRACKET_SPACING_OPTION = {
 	]
 };
 
+export const PUG_ARROW_PARENS_OPTION = {
+	since: '1.6.2',
+	category: CATEGORY_PUG,
+	type: 'choice',
+	default: null,
+	choices: [
```

Comment:
You should copy the descriptions defined by prettier itself: https://github.com/prettier/prettier/blob/6acf5370f7154ac66cd0db270c71b60e5e136eca/src/language-js/options.js#L9-L28
_all 3_

Also swap both objects, I don't know if this will have an effect on CLI suggestions

Reply:
Ah, i was looking for this but couldn't find it. Perfect!

Reply:
What do you mean with _all 3_?

Reply:
Ah, `null` needs to be included also, go tit.